src/main/target/common_post.h

```diff
@@ -214,3 +214,7 @@
 #define SYSTEM_HSE_VALUE (HSE_VALUE/1000000U)
 #endif
 #endif
+
+#if (defined(USE_FLASHFS) && defined(USE_RTC_TIME) && defined(USE_USB_MSC))
+#define PERSIST_MSC_RTC
```

Comment:
Can I suggest we do this the other way round? I.e. define `PERSIST_MSC_RTC` in `common_pre.h` for MCU types we want it in, and then undefine in `common_post.h` if any of the prerequisites are not defined? This way, individual targets can undefine it explicitly if there's a reason to do so.

Reply:
Makes perfect sense. Any preferences on where in `common_pre.h` the `#define PERSIST_MSC_RTC` lives? Could either be in the `#if (FLASH_SIZE > 256)` section or in the `#ifdef STM32F4` and `#ifdef STM32F7` sections. I guessing probably the latter as that's also where `USE_USB_MSC` is defined - both either way would work.  I guess I should also rename it if we're going this way to something like `USE_PERSISTENT_MSC_RTC`.

Reply:
The F4 / F7 sections are probably the best - anything that is dependent on MCU type (and not just a question of firmware size) should go in there.

Reply:
Ammended as suggested. Renamed to `USE_PERSISTENT_MSC_RTC`.